### PR TITLE
Add final touch ups

### LINKED ISSUE #100: Update Userdocs content
Currently, user guide still uses the AB3 content. 

I will simply write for the new commands we added and fit it into the existing structure.

### PR DESCRIPTION
**Pull request by Quasant**
Deleted artefacts that are not needed, or not touched. 

I removed a bunch of To-do's, so do take note of them. if you wanna do a last min sprint.

Also modified the UG to reflect some changes.

Closes #100

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -373,5 +373,5 @@ _Details coming soon ..._
 | **Find**           | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`                                                                                                                       |
 | **Help**           | `help`                                                                                                                                                                           |
 | **Exit**           | `exit`                                                                                                                                                                           |
-| **Stats**          | `stats`                                                                                                                                                                                  |
+| **Stats**          | `stats`                                                                                                                                                                          |
```
Comment (Quasant) [inline docs/UserGuide.md:376]: wait ihni why this was recorded as a change

**Review by Green-Tea-123 — APPROVED** [review]
LGTM.

File: docs/UserGuide.md (hunk 1 above)
Comment (Green-Tea-123) [reply docs/UserGuide.md:376]: actually can you add in the contact for the find and edit command?

Comment (Quasant) [reply docs/UserGuide.md:376]: Done, I just changed this from draft to ready for review, so take a look.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -323,7 +323,7 @@ AddressBook data are saved in the hard disk automatically after any command that
 ### Editing the data file
 
 AddressBook data are saved automatically as a JSON file:<br>
-`[JAR file location]/data/addressbook.json`.<br>
+`[JAR file location]/data/talentconnect.json`.<br>
```
Comment (AlekKwek18) [inline docs/UserGuide.md:326]: might have merge conflict at this line.

**Review by AlekKwek18 — APPROVED** [review]
Overall LGTM

**Pull request merged**